[2025-06-26T03:27:24.155+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-26T03:27:24.209+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment scheduled__2025-06-25T03:00:00+00:00 [queued]>
[2025-06-26T03:27:24.227+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment scheduled__2025-06-25T03:00:00+00:00 [queued]>
[2025-06-26T03:27:24.229+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-06-26T03:27:24.254+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): validate_environment> on 2025-06-25 03:00:00+00:00
[2025-06-26T03:27:24.267+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=5258) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-26T03:27:24.268+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'validate_environment', 'scheduled__2025-06-25T03:00:00+00:00', '--job-id', '294', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmp319kyym4']
[2025-06-26T03:27:24.273+0000] {standard_task_runner.py:72} INFO - Started process 5259 to run task
[2025-06-26T03:27:24.273+0000] {standard_task_runner.py:105} INFO - Job 294: Subtask validate_environment
[2025-06-26T03:27:24.467+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.validate_environment scheduled__2025-06-25T03:00:00+00:00 [running]> on host 003a60651790
[2025-06-26T03:27:24.643+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Jasjot Parmar' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='validate_environment' AIRFLOW_CTX_EXECUTION_DATE='2025-06-25T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-25T03:00:00+00:00'
[2025-06-26T03:27:24.646+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-06-26T03:27:24.701+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-06-26T03:27:24.708+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', ' \n        echo "Validating environment..."\n        test -f /app/config.yaml &&\n        test -f /app/.env &&\n        echo "Environment is valid!"\n        ']
[2025-06-26T03:27:24.733+0000] {subprocess.py:99} INFO - Output:
[2025-06-26T03:27:24.769+0000] {subprocess.py:106} INFO - Validating environment...
[2025-06-26T03:27:24.773+0000] {subprocess.py:106} INFO - Environment is valid!
[2025-06-26T03:27:24.774+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-06-26T03:27:24.831+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-06-26T03:27:24.832+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=validate_environment, run_id=scheduled__2025-06-25T03:00:00+00:00, execution_date=20250625T030000, start_date=20250626T032724, end_date=20250626T032724
[2025-06-26T03:27:24.922+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-26T03:27:24.976+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-26T03:27:24.979+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
